plugins/helo.checks.js

```diff
@@ -170,7 +170,7 @@ exports.valid_hostname = function (next, connection, helo) {
     if (!/\./.test(helo)) {
         connection.results.add(plugin, {fail: 'valid_hostname(no_dot)'});
         if (plugin.cfg.reject.valid_hostname) {
-            return next(DENY, 'Host names have more than one DNS label');
+            return next(DENY, 'HELO host name must have a dot');
```

Comment:
This is a SMTP denial/rejection message, not an error message.

I also don't agree this is a better message.

````
   Local nicknames or
   unqualified names MUST NOT be used.  There are two exceptions to the
   rule requiring FQDNs:

   o  The domain name given in the EHLO command MUST be either a primary
      host name (a domain name that resolves to an address RR) or, if
      the host has no name, an address literal, as described in
      Section 4.1.3 and discussed further in the EHLO discussion of
      Section 4.1.4.
````

While a primary domain name / FQDN and address literals both do have dots, that's a proxy for the requirements and not a very useful rejection message. How about this instead:

````
return next(DENY, 'HELO host must be a FQDN or address literal (RFC 5321 §2.3.5');
````